tools/ctl/rundex/rundex.go

```diff
@@ -245,7 +245,7 @@ func filterRebuilds(all <-chan Rebuild, req *FetchRebuildRequest) map[string]Reb
 	}
 	if req.Opts.Filter != "" {
```

Comment:
changing --filter to --matches-prefix at the cli could maintain the current behavior while we could add --matches for Contains (or, potentially, a full regex).

Reply:
Changed --filter to --prefix and the new behavior is --pattern.